Godeps/Godeps.json

```diff
@@ -1351,132 +1351,132 @@
 		},
 		{
 			"ImportPath": "github.com/docker/docker/api",
-			"Comment": "docs-v1.12.0-rc4-2016-07-15-9510-ga9fbbdc8d",
+			"Comment": "docs-v1.12.0-rc4-2016-07-15-9510-ga9fbbdc8dd",
```

Comment:
Also @ixdy do you know why the extra `d` is getting appended here? I have a vendor change that is doing the same thing and I don't know why.

Reply:
I have no idea how `godep` decides how to truncate the git commit. it might be a git setting, it might be something else. `godep` is terrible.

Reply:
I think it's just using `git describe --tags`. 

The manpage for `git describe` notes
> Note that the suffix you get if you type these commands today may be longer than what Linus saw  above when he ran these commands, as your Git repository may have new commits whose object names begin with 975b that did not exist back then, and "-g975b" suffix alone may not be sufficient to disambiguate these commits.

so I guess this is WAI? :man_shrugging:

Reply:
I remember discussing about this with @cblecker somewhere but can't find the issue now... :woman_shrugging:

Reply:
it has to do with the git version you're running locally. at a certain point, the default changed for `git describe`

see https://github.com/tools/godep/pull/560 for more details. This PR to fix it was opened, but that happened after godep stopped active development.

Reply:
Similar issue raised in another PR: https://github.com/kubernetes/kubernetes/pull/65632#discussion_r227509437

What is the course of action here? Just ignore these changes? Manually revert them?

Reply:
+1 to not include the changes that update the comment with just one more character or one less character for the SHA

Reply:
We could also just scrub all comments from the file. That's what we do with the staging-godeps script

Reply:
We probably should.  This is getting stupid.

Reply:
+1 to scrub all comments from the file